worker/src/main.rs

```diff
@@ -861,87 +728,53 @@ pub unsafe extern "C" fn ocall_send_release_extrinsic(
             vec![]
         }
     };
-    api.send_extrinsic(hex_encode(release_extrinsic_calls), XtStatus::Ready)
+    api.send_extrinsic(release_extrinsic_calls, XtStatus::Ready)
         .unwrap();
     status
 }
 
-/// # Safety
-///
-/// FFI are always unsafe
-#[no_mangle]
-pub unsafe extern "C" fn ocall_send_block_and_confirmation(
-    confirmations: *const u8,
```

Comment:
Quite a git diff mess here.
Effective change: `ocall_send_block_and_confirmation` has been moved to ffi.rs, `get_blocks_to_sync` fn has been created (copy paste of the code above, that has been removed from https://github.com/Polkadex-Substrate/polkadexTEE-worker/pull/288/files?authenticity_token=w7eCHmslS9aK5g%2BUgs9Yj7rPTeprNyHXTABTmRTZhKu8n5VuyWB%2FU8XlOXVuKVGTq8s3azdnk%2B9Td%2FpH0VATUg%3D%3D&file-filters%5B%5D=.bin&file-filters%5B%5D=.der&file-filters%5B%5D=.pyc&file-filters%5B%5D=.rs&hide-deleted-files=true&w=1#diff-3381a66150dcfc521726bae16533dcb6463f5f799ba999a2f972bee40756a6bbL606-L637